src/components/recipes/RecipeForm.js

```diff
@@ -69,7 +69,10 @@ export default class RecipeForm extends React.PureComponent {
   render() {
     const { isCreationForm, isLoading, onSubmit, recipe, selectedActionName } = this.props;
 
-    const ArgumentsFields = RecipeForm.argumentsFields[selectedActionName];
+    let ArgumentsFields = null;
```

Comment:
nit: you could just leave this all unchanged and it would still fix the issue.

Reply:
I just thought it ugly to rely on something that might be `undefined`.

Reply:
ah, I would've done this then. which actually ensure it's never `undefined`:
```js
const ArgumentsFields = RecipeForm.argumentsField[selectedActionName] || null;
```

Reply:
No, what I meant was that I didn't want to *use* `undefined`. I.e. this: `RecipeForm.argumentsField[undefined]`.

Reply:
oh. it would not be undefined. it would be `RecipeForm.argumentsField[null]`. I suppose that's not much different. But it seems like we made things more verbose for no reason.